torchvision/prototype/transforms/functional/_color.py

```diff
@@ -119,10 +181,31 @@ def autocontrast(inpt: features.InputTypeJIT) -> features.InputTypeJIT:
         return autocontrast_image_pil(inpt)
 
 
-equalize_image_tensor = _FT.equalize
+def equalize_image_tensor(image: torch.Tensor) -> torch.Tensor:
+    _FT._assert_image_tensor(image)
+
+    if image.dtype != torch.uint8:
+        raise TypeError(f"Only torch.uint8 image tensors are supported, but found {image.dtype}")
+
+    _FT._assert_channels(image, [1, 3])
+
+    if image.numel() == 0:
+        return image
+    elif image.ndim == 2:
+        return _FT._scale_channel(image)
+    else:
+        shape = image.shape
+        channels = image.view((-1,) + shape[-2:]).unbind()
+        return torch.stack([_FT._scale_channel(channel) for channel in channels]).view(shape)
```

Comment:
Why not?
```suggestion
        return _FT._equalize_single_image(channels).view(shape)
```

Reply:
I wanted to avoid that someone copy-pastes `_equalize_single_image` as it can easily be inlined. I'll add a comment and use it here.